portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,29 +27,45 @@
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.Sentiment;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
+//An object that will be converted to JSON. Contains comments and their sentiment scores
```

Comment:
For class comments, it's good to use javadoc formatting (the ./** style you see right above this comment you made)

Reply:
Got it, made some changes to the comments. They should be good now.